### PR TITLE
add ci yaml for checking the build

### PR DESCRIPTION
**Pull request by yolandaz**
tested by introducing a change that breaks the deploy and making sure the build fails ([PR](https://github.com/stephen/static/pull/7))
image

### CODE REVIEW

**Comment by yolandaz** [comment]
@stephen can i have settings access so i can make this check mandatory to merge

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          cp -r ./src/. out/
```
Comment (stephen) [inline .github/workflows/ci.yml:17]: no

**Review by stephen — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yolandaz) [reply .github/workflows/ci.yml:17]: fine

Comment (yolandaz) [reply .github/workflows/ci.yml]: ok i refactored it

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          chmod +x ./bigfuncloud/build.sh
```
Comment (stephen) [inline .github/workflows/ci.yml:17]: do this once and check it in

**Review by stephen — APPROVED** [review]

**Pull request merged**